.github/workflows/create-jira-ticket.yml

```
@@ -0,0 +1,56 @@
+          ADF_JSON=$(echo "${{ github.event.issue.body }}" | md-to-adf)
+          ADF_JSON_ESCAPED=$(echo "$ADF_JSON" | jq -c .)
+          echo "adf=$ADF_JSON_ESCAPED" >> $GITHUB_OUTPUT
+
+      - name: Create JIRA ticket
+        if: steps.check_label.outputs.has_label == 'true'
+        env:
+          JIRA_USER: ${{ secrets.JIRA_USER }}
+          JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
+          JIRA_PROJECT: ${{ secrets.JIRA_PROJECT }}
+          JIRA_URL: ${{ secrets.JIRA_URL }}
+        run: |
```

actionlint (1.7.8)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)